templates/boundary_custom_data.sh.tpl

```diff
@@ -13,7 +13,11 @@ BOUNDARY_DIR_LOGS="/var/log/boundary"
 BOUNDARY_DIR_BIN="${boundary_dir_bin}"
 BOUNDARY_USER="boundary"
 BOUNDARY_GROUP="boundary"
-BOUNDARY_INSTALL_URL="${boundary_install_url}"
+# BOUNDARY_INSTALL_URL="${boundary_install_url}"
+PRODUCT="boundary"
+OS_ARCH="linux_$( dpkg --print-architecture )"
```

Comment:
Sorry, same as the Nomad one, dpkg will only work on Debian derivatives, we'll need an alternative for other distros.